script/metric/step_3_extract_metric_values.rb

```diff
@@ -92,8 +92,10 @@ def get_aliases_hash
         end
         if _company != nil
           metric_name = accepted_snippet_name["#{source_name}+#{name}"]
-          result["\"#{metric_name}\",\"#{_company}\",\"#{value["citeyear"]}\""] =
-           "\"#{_value}\",\"#{source_link}\""
+          if !Card.exists?("#{metric_name}+#{_company}+#{value['citeyear']}")
+            result["\"#{metric_name}\",\"#{_company}\",\"#{value["citeyear"]}\""] =
```

Comment:
Line is too long. [83/80]<br>Prefer single-quoted strings inside interpolations.